spec/cell_spec.rb

```diff
@@ -71,14 +71,31 @@
     
     it 'will update ship health if cell is occupied' do
       @cell.place_ship(@cruiser)
+      expect(@cell.fired_upon).to eq(false)
       @cell.fire_upon
       
       expect(@cell.ship.health).to eq(2)
+      expect(@cell.fired_upon).to eq(true)
     end
   
 end
 
   describe "#render()" do
+    it 'will return the string "." if the cell has not been fired upon' do
```

Comment:
I like that your describe blocks are the names of the methods, I like how you are grouping the tests. I will work on that so I am following your setup.